src/lib.rs

```diff
@@ -81,14 +74,47 @@ impl Game {
         Ok(())
     }
 
+    fn simulate_internal(&mut self, elapsed: f64, inputs: InputState) {
+        // elapsed is the time, in milliseconds, that has passed since the
+        // last time we simulated.
+        // By making our simulations relative to the amount of time that's passed,
+        // the game will progress the same regardless of the frame rate, which may
+        // vary between 30fps and 144fps even if our performance is perfect!
```

Comment:
This might make for an interesting read https://gafferongames.com/post/fix_your_timestep/

Reply:
> This interpolation is easy to do for single values and for vector state values. You can even use it with full 3D rigid body dynamics if you store your orientation as a quaternion and use a spherical linear interpolation (slerp) to blend between the previous and current orientations.

D: